packages/css/src/components/password-input/README.md

```diff
@@ -0,0 +1,25 @@
+<!-- @license CC0-1.0 -->
+
+# Password Input
+
+Helps users enter a password.
+
+Use this component when the input requires sensitive information, like passwords or PINs.
+It ensures that the input is not readable by others who might be looking at the screen.
+The characters entered are hidden, represented by squares.
+
+Consider setting the following attributes:
```

Comment:
These are guidelines, no? Why the different format?

Reply:
Copied from Text Input, where it was an example. Adjusted.